app/templates/core/contact.pug

```diff
@@ -7,12 +7,16 @@ block modal-body-content
   p
     span(data-i18n="contact.welcome")
     span.spl(data-i18n="contact.forum_prefix")
-    a(href="https://discourse.codecombat.com/", data-i18n="contact.forum_page")
-    span(data-i18n="contact.forum_suffix")
-    span.spl.spr(data-i18n="contact.faq_prefix")
+    if me.isHomeUser()
+      a(href="https://discourse.codecombat.com/", data-i18n="contact.forum_page")
+      span(data-i18n="contact.forum_suffix")
```

Comment:
I can't find this i18n, how does it work? I know you haven't added it, just curious

Reply:
lol seems it just empty. but ya strange, non-existed i18n should report error but this is not

Reply:
![image](https://github.com/codecombat/codecombat/assets/11417632/494aa567-57ee-4339-94d8-784e28c791ba)